run.py

```diff
@@ -14,19 +13,20 @@ def run():
         url=video_url, download=False
     )
 
-    vedio_title = video_info['title']
-    filename = f"{vedio_title}.mp3"
+    video_title = video_info['title']
+    filename = f"{video_title}.mp3"
 
     # Ask the user for the path they want to save the file after it has been downloaded
     path_to_save = input(
         r"Enter where the file is to be saved or leave empty if you want it to be saved in the current directory: ")
+
+    where_to_save = f"{path_to_save}"
+
     if path_to_save == "":
```

Comment:
If you add the above, here you would do `if path_to_save != ""` to check that it's not empty. If it is not, then you would have `where_to_save = f"{path_to_save}"`?

Reply:
I think they pretty much do the same thing. But your suggestion looks more readable and easier to follow logically. Worth adopting! 😉